packages/replayio/src/commands/record.ts

```diff
@@ -102,11 +108,20 @@ async function record(url: string = "about:blank") {
     }
 
     console.log(""); // Spacing for readability
+  } else if (unusableRecordings.length > 0) {
+    // If there were unusable recordings we should provide explicit messaging about them
+    const reason = unusableRecordings.findLast(
+      recording => recording.recordingStatus === "unusable" && recording.unusableReason
+    )?.unusableReason;
```

Comment:
We are already working on the filtered list of unusable recordings so rechecking that `recordingStatus` is redundant.

```suggestion
    const reason = unusableRecordings.findLast(recording => !!recording.unusableReason)?.unusableReason;
```